tests/components/trafikverket_weatherstation/test_config_flow.py

```diff
@@ -192,3 +192,104 @@ async def test_reauth_flow_fails(
 
     assert result["type"] is FlowResultType.FORM
     assert result["errors"] == {"base": base_error}
+
+
+async def test_reconfigure_flow(hass: HomeAssistant) -> None:
+    """Test a reconfigure flow."""
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_API_KEY: "1234567890",
+            CONF_STATION: "Vallby",
+        },
+    )
+    entry.add_to_hass(hass)
+
+    result = await hass.config_entries.flow.async_init(
+        entry.domain,
+        context={
+            "source": config_entries.SOURCE_RECONFIGURE,
+            "entry_id": entry.entry_id,
+        },
+    )
```

Comment:
```suggestion
    result = await entry.start_reconfigure_flow(hass)
```

Reply:
I thought I looked for it but didn't find it 👍

Reply:
> I thought I looked for it but didn't find it 👍

Brand new: #127154

Reply:
So not fair then ;)
I will mod it.